pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -322,6 +331,25 @@ func (b *OFBridge) MultipartReply(sw *ofctrl.OFSwitch, rep *openflow15.Multipart
 	if ch, ok := b.multipartReplyChs[rep.Xid]; ok {
 		ch <- rep
 	}
+	if rep.Type == openflow15.MultipartType_MeterStats {
+		statsArr := rep.Body
+		for _, entry := range statsArr {
+			stats := entry.(*openflow15.MeterStats)
+			idLabel := ""
+			if stats.MeterId == 1 {
+				idLabel = "PacketInMeterIDNP"
+			} else if stats.MeterId == 2 {
+				idLabel = "PacketInMeterIDTF"
```

Comment:
these labels are exposed to the user, so please make them more user-friendly, e.g. "PacketInMeterNetworkPolicy" and "PacketInMeterTraceflow".

Reply:
Thanks, done